lib/uploader.ts

```diff
@@ -201,7 +201,18 @@ export class Uploader {
 
 					// Init request queue
 					const request = () => {
-						return uploadData(`${tempUrl}/${chunk+1}`, blob, upload.signal, () => this.updateStats(), destinationFile)
+						return uploadData(
+							`${tempUrl}/${chunk+1}`,
+							blob,
+							upload.signal,
+							() => this.updateStats(),
+							destinationFile,
+							{
+								'X-OC-Mtime': file.lastModified,
```

Comment:
There is also a `X-OC-CTime` header that could be used if we can read the actual file creation time from the file object.

Reply:
We can add it once we know how to get it